### ISSUE TITLE
[IntelliJ][ReportedByUser] Run into exception when link an Aris cluster

### ISSUE THREAD

**Issue by jingyanjingyan** [microsoft/azure-tools-for-java#3831]
Build:
azure-toolkit-for-intellij-2019.2.4.develop.1317.12-10-2019.zip

Repro Steps:
1. Try to link aris cluster

Result:
Error.

IntelliJ build version: 2019.2.4 IC-192.7142.36
OS: Windows 10
JDK: JetBrains s.r.o 11.0.4
Plugin version: 3.30.0
Additional Info: None
Parent component: 
```
com.intellij.openapi.wm.impl.IdeRootPane[,8,8,1441x819,invalid,layout=com.intellij.openapi.wm.impl.IdeRootPane$MyRootLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=449,maximumSize=,minimumSize=,preferredSize=]
```
Error message: 
```
java.lang.ClassCastException: class java.lang.String cannot be cast to class com.microsoft.azure.hdinsight.sdk.common.AuthType (java.lang.String is in module java.base of loader 'bootstrap'; com.microsoft.azure.hdinsight.sdk.common.AuthType is in unnamed module of loader com.intellij.ide.plugins.cl.PluginClassLoader @42741143)
	at com.microsoft.azure.hdinsight.serverexplore.ui.AddNewClusterForm$1.customize(AddNewClusterForm.java:152)
	at com.intellij.ui.SimpleListCellRenderer.getListCellRendererComponent(SimpleListCellRenderer.java:64)
	at java.desktop/javax.swing.plaf.basic.BasicComboBoxUI.getDisplaySize(BasicComboBoxUI.java:1407)
	at java.desktop/javax.swing.plaf.basic.BasicComboBoxUI.getMinimumSize(BasicComboBoxUI.java:960)
	at com.intellij.ide.ui.laf.darcula.ui.DarculaComboBoxUI.getMinimumSize(DarculaComboBoxUI.java:518)
	at java.desktop/javax.swing.JComponent.getMinimumSize(JComponent.java:1760)
	at com.intellij.uiDesigner.core.Util.getMinimumSize(Util.java:27)
	at com.intellij.uiDesigner.core.DimensionInfo.getMinimumSize(DimensionInfo.java:239)
	at com.intellij.uiDesigner.core.HorizontalInfo.getMinimumWidth(HorizontalInfo.java:40)
	at com.intellij.uiDesigner.core.GridLayoutManager.getMin2(GridLayoutManager.java:739)
	at com.intellij.uiDesigner.core.GridLayoutManager.getMinOrPrefSizes(GridLayoutManager.java:645)
	at com.intellij.uiDesigner.core.GridLayoutManager.getMinSizes(GridLayoutManager.java:626)
	at com.intellij.uiDesigner.core.GridLayoutManager.minimumLayoutSize(GridLayoutManager.java:221)
	at java.desktop/java.awt.Container.minimumSize(Container.java:1875)
	at java.desktop/java.awt.Container.getMinimumSize(Container.java:1859)
	at java.desktop/javax.swing.JComponent.getMinimumSize(JComponent.java:1762)
	at com.intellij.uiDesigner.core.Util.getMinimumSize(Util.java:27)
	at com.intellij.uiDesigner.core.DimensionInfo.getMinimumSize(DimensionInfo.java:239)
	at com.intellij.uiDesigner.core.HorizontalInfo.getMinimumWidth(HorizontalInfo.java:40)
	at com.intellij.uiDesigner.core.GridLayoutManager.getMin2(GridLayoutManager.java:734)
	at com.intellij.uiDesigner.core.GridLayoutManager.getMinOrPrefSizes(GridLayoutManager.java:645)
	at com.intellij.uiDesigner.core.GridLayoutManager.getMinSizes(GridLayoutManager.java:626)
	at com.intellij.uiDesigner.core.GridLayoutManager.minimumLayoutSize(GridLayoutManager.java:221)
	at java.desktop/java.awt.Container.minimumSize(Container.java:1875)
	at java.desktop/java.awt.Container.getMinimumSize(Container.java:1859)
	at java.desktop/javax.swing.JComponent.getMinimumSize(JComponent.java:1762)
	at com.intellij.uiDesigner.core.Util.getMinimumSize(Util.java:27)
	at com.intellij.uiDesigner.core.DimensionInfo.getMinimumSize(DimensionInfo.java:239)
	at com.intellij.uiDesigner.core.HorizontalInfo.getMinimumWidth(HorizontalInfo.java:40)
	at com.intellij.uiDesigner.core.GridLayoutManager.getMinOrPrefSizes(GridLayoutManager.java:645)
	at com.intellij.uiDesigner.core.GridLayoutManager.getPrefSizes(GridLayoutManager.java:630)
	at com.intellij.uiDesigner.core.GridLayoutManager.preferredLayoutSize(GridLayoutManager.java:274)
	at java.desktop/java.awt.Container.preferredSize(Container.java:1823)
	at java.desktop/java.awt.Container.getPreferredSize(Container.java:1807)
	at java.desktop/javax.swing.JComponent.getPreferredSize(JComponent.java:1682)
	at java.desktop/java.awt.BorderLayout.preferredLayoutSize(BorderLayout.java:724)
	at java.desktop/java.awt.Container.preferredSize(Container.java:1823)
	at java.desktop/java.awt.Container.getPreferredSize(Container.java:1807)
	at java.desktop/javax.swing.JComponent.getPreferredSize(JComponent.java:1682)
	at java.desktop/java.awt.BorderLayout.preferredLayoutSize(BorderLayout.java:724)
	at java.desktop/java.awt.Container.preferredSize(Container.java:1823)
	at java.desktop/java.awt.Container.getPreferredSize(Container.java:1807)
	at java.desktop/javax.swing.JComponent.getPreferredSize(JComponent.java:1682)
	at java.desktop/java.awt.BorderLayout.preferredLayoutSize(BorderLayout.java:724)
	at java.desktop/java.awt.Container.preferredSize(Container.java:1823)
	at java.desktop/java.awt.Container.getPreferredSize(Container.java:1807)
	at java.desktop/javax.swing.JComponent.getPreferredSize(JComponent.java:1682)
	at net.miginfocom.swing.SwingComponentWrapper.getLayoutHashCode(Unknown Source)
	at net.miginfocom.swing.MigLayout.checkCache(Unknown Source)
	at net.miginfocom.swing.MigLayout.getSizeImpl(Unknown Source)
	at net.miginfocom.swing.MigLayout.preferredLayoutSize(Unknown Source)
	at java.desktop/java.awt.Container.preferredSize(Container.java:1823)
	at java.desktop/java.awt.Container.getPreferredSize(Container.java:1807)
	at java.desktop/javax.swing.JComponent.getPreferredSize(JComponent.java:1682)
	at java.desktop/javax.swing.JRootPane$RootLayout.preferredLayoutSize(JRootPane.java:866)
	at java.desktop/java.awt.Container.preferredSize(Container.java:1823)
	at java.desktop/java.awt.Container.getPreferredSize(Container.java:1807)
	at java.desktop/javax.swing.JComponent.getPreferredSize(JComponent.java:1682)
	at java.desktop/java.awt.BorderLayout.preferredLayoutSize(BorderLayout.java:724)
	at java.desktop/java.awt.Container.preferredSize(Container.java:1823)
	at java.desktop/java.awt.Container.getPreferredSize(Container.java:1807)
	at java.desktop/java.awt.Window.pack(Window.java:826)
	at com.intellij.openapi.ui.impl.DialogWrapperPeerImpl$MyDialog.show(DialogWrapperPeerImpl.java:648)
	at com.intellij.openapi.ui.impl.DialogWrapperPeerImpl.show(DialogWrapperPeerImpl.java:432)
	at com.intellij.openapi.ui.DialogWrapper.doShow(DialogWrapper.java:1685)
	at com.intellij.openapi.ui.DialogWrapper.show(DialogWrapper.java:1644)
	at com.microsoft.sqlbigdata.serverexplore.action.LinkSqlServerBigDataClusterAction.actionPerformed(LinkSqlServerBigDataClusterAction.kt:36)
	at com.microsoft.tooling.msservices.serviceexplorer.NodeActionListener.actionPerformedAsync(NodeActionListener.java:79)
	at com.microsoft.tooling.msservices.serviceexplorer.NodeAction.fireNodeActionEvent(NodeAction.java:60)
	at com.microsoft.intellij.components.ServerExplorerToolWindowFactory.lambda$createPopupMenuForNode$1(ServerExplorerToolWindowFactory.java:219)
	at java.desktop/javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:1967)
	at java.desktop/javax.swing.AbstractButton$Handler.actionPerformed(AbstractButton.java:2308)
	at java.desktop/javax.swing.DefaultButtonModel.fireActionPerformed(DefaultButtonModel.java:405)
	at java.desktop/javax.swing.DefaultButtonModel.setPressed(DefaultButtonModel.java:262)
	at java.desktop/javax.swing.AbstractButton.doClick(AbstractButton.java:369)
	at java.desktop/javax.swing.plaf.basic.BasicMenuItemUI.doClick(BasicMenuItemUI.java:980)
	at java.desktop/javax.swing.plaf.basic.BasicMenuItemUI$Handler.mouseReleased(BasicMenuItemUI.java:1024)
	at java.desktop/java.awt.Component.processMouseEvent(Component.java:6651)
	at java.desktop/javax.swing.JComponent.processMouseEvent(JComponent.java:3342)
	at java.desktop/java.awt.Component.processEvent(Component.java:6416)
	at java.desktop/java.awt.Container.processEvent(Container.java:2263)
	at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:5026)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2321)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4858)
	at java.desktop/java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4918)
	at java.desktop/java.awt.LightweightDispatcher.processMouseEvent(Container.java:4547)
	at java.desktop/java.awt.LightweightDispatcher.dispatchEvent(Container.java:4488)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2307)
	at java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2773)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4858)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:778)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:727)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:95)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:751)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:749)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:748)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:878)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:823)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:466)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:704)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:465)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)

**Comment by jingyanjingyan** [comment]
Link cluster in reader role warning dialog get error:
java.lang.ClassCastException: java.lang.String cannot be cast to com.microsoft.azure.hdinsight.sdk.common.AuthType
	at com.microsoft.azure.hdinsight.serverexplore.ui.AddNewClusterForm$1.customize(AddNewClusterForm.java:152)
	at com.intellij.ui.SimpleListCellRenderer.getListCellRendererComponent(SimpleListCellRenderer.java:64)
	at javax.swing.plaf.basic.BasicComboBoxUI.getDisplaySize(BasicComboBoxUI.java:1353)
	at javax.swing.plaf.basic.BasicComboBoxUI.getMinimumSize(BasicComboBoxUI.java:921)
	at com.intellij.ide.ui.laf.darcula.ui.DarculaComboBoxUI.getMinimumSize(DarculaComboBoxUI.java:518)
	at javax.swing.JComponent.getMinimumSize(JComponent.java:1745)
	at com.intellij.uiDesigner.core.Util.getMinimumSize(Util.java:27)
	at com.intellij.uiDesigner.core.DimensionInfo.getMinimumSize(DimensionInfo.java:239)
	at com.intellij.uiDesigner.core.HorizontalInfo.getMinimumWidth(HorizontalInfo.java:40)
	at com.intellij.uiDesigner.core.GridLayoutManager.getMin2(GridLayoutManager.java:739)
	at com.intellij.uiDesigner.core.GridLayoutManager.getMinOrPrefSizes(GridLayoutManager.java:645)
	at com.intellij.uiDesigner.core.GridLayoutManager.getMinSizes(GridLayoutManager.java:626)
	at com.intellij.uiDesigner.core.GridLayoutManager.minimumLayoutSize(GridLayoutManager.java:221)
	at java.awt.Container.minimumSize(Container.java:1851)
	at java.awt.Container.getMinimumSize(Container.java:1835)
	at javax.swing.JComponent.getMinimumSize(JComponent.java:1747)
	at com.intellij.uiDesigner.core.Util.getMinimumSize(Util.java:27)
	at com.intellij.uiDesigner.core.DimensionInfo.getMinimumSize(DimensionInfo.java:239)
	at com.intellij.uiDesigner.core.HorizontalInfo.getMinimumWidth(HorizontalInfo.java:40)
	at com.intellij.uiDesigner.core.GridLayoutManager.getMin2(GridLayoutManager.java:734)
	at com.intellij.uiDesigner.core.GridLayoutManager.getMinOrPrefSizes(GridLayoutManager.java:645)
	at com.intellij.uiDesigner.core.GridLayoutManager.getMinSizes(GridLayoutManager.java:626)
	at com.intellij.uiDesigner.core.GridLayoutManager.minimumLayoutSize(GridLayoutManager.java:221)
	at java.awt.Container.minimumSize(Container.java:1851)
	at java.awt.Container.getMinimumSize(Container.java:1835)
	at javax.swing.JComponent.getMinimumSize(JComponent.java:1747)
	at com.intellij.uiDesigner.core.Util.getMinimumSize(Util.java:27)
	at com.intellij.uiDesigner.core.DimensionInfo.getMinimumSize(DimensionInfo.java:239)
	at com.intellij.uiDesigner.core.HorizontalInfo.getMinimumWidth(HorizontalInfo.java:40)
	at com.intellij.uiDesigner.core.GridLayoutManager.getMinOrPrefSizes(GridLayoutManager.java:645)
	at com.intellij.uiDesigner.core.GridLayoutManager.getPrefSizes(GridLayoutManager.java:630)
	at com.intellij.uiDesigner.core.GridLayoutManager.preferredLayoutSize(GridLayoutManager.java:274)
	at java.awt.Container.preferredSize(Container.java:1799)
	at java.awt.Container.getPreferredSize(Container.java:1783)
	at javax.swing.JComponent.getPreferredSize(JComponent.java:1665)
	at java.awt.BorderLayout.preferredLayoutSize(BorderLayout.java:719)
	at java.awt.Container.preferredSize(Container.java:1799)
	at java.awt.Container.getPreferredSize(Container.java:1783)
	at javax.swing.JComponent.getPreferredSize(JComponent.java:1665)
	at java.awt.BorderLayout.preferredLayoutSize(BorderLayout.java:719)
	at java.awt.Container.preferredSize(Container.java:1799)
	at java.awt.Container.getPreferredSize(Container.java:1783)
	at javax.swing.JComponent.getPreferredSize(JComponent.java:1665)
	at java.awt.BorderLayout.preferredLayoutSize(BorderLayout.java:719)
	at java.awt.Container.preferredSize(Container.java:1799)
	at java.awt.Container.getPreferredSize(Container.java:1783)
	at javax.swing.JComponent.getPreferredSize(JComponent.java:1665)
	at net.miginfocom.swing.SwingComponentWrapper.getLayoutHashCode(Unknown Source)
	at net.miginfocom.swing.MigLayout.checkCache(Unknown Source)
	at net.miginfocom.swing.MigLayout.getSizeImpl(Unknown Source)
	at net.miginfocom.swing.MigLayout.preferredLayoutSize(Unknown Source)
	at java.awt.Container.preferredSize(Container.java:1799)
	at java.awt.Container.getPreferredSize(Container.java:1783)
	at javax.swing.JComponent.getPreferredSize(JComponent.java:1665)
	at javax.swing.JRootPane$RootLayout.preferredLayoutSize(JRootPane.java:920)
	at java.awt.Container.preferredSize(Container.java:1799)
	at java.awt.Container.getPreferredSize(Container.java:1783)
	at javax.swing.JComponent.getPreferredSize(JComponent.java:1665)
	at java.awt.BorderLayout.preferredLayoutSize(BorderLayout.java:719)
	at java.awt.Container.preferredSize(Container.java:1799)
	at java.awt.Container.getPreferredSize(Container.java:1783)
	at java.awt.Window.pack(Window.java:809)
	at com.intellij.openapi.ui.impl.DialogWrapperPeerImpl$MyDialog.show(DialogWrapperPeerImpl.java:648)
	at com.intellij.openapi.ui.impl.DialogWrapperPeerImpl.show(DialogWrapperPeerImpl.java:432)
	at com.intellij.openapi.ui.DialogWrapper.doShow(DialogWrapper.java:1685)
	at com.intellij.openapi.ui.DialogWrapper.show(DialogWrapper.java:1644)
	at com.microsoft.azure.hdinsight.common.HDInsightHelperImpl$4$1.doOKAction(HDInsightHelperImpl.java:210)
	at com.intellij.openapi.ui.DialogWrapper$OkAction.doAction(DialogWrapper.java:1861)
	at com.intellij.openapi.ui.DialogWrapper$DialogWrapperAction.actionPerformed(DialogWrapper.java:1821)
	at javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:2022)
	at javax.swing.AbstractButton$Handler.actionPerformed(AbstractButton.java:2348)
	at javax.swing.DefaultButtonModel.fireActionPerformed(DefaultButtonModel.java:402)
	at javax.swing.DefaultButtonModel.setPressed(DefaultButtonModel.java:259)
	at javax.swing.plaf.basic.BasicButtonListener.mouseReleased(BasicButtonListener.java:252)
	at java.awt.Component.processMouseEvent(Component.java:6550)
	at javax.swing.JComponent.processMouseEvent(JComponent.java:3325)
	at java.awt.Component.processEvent(Component.java:6315)
	at java.awt.Container.processEvent(Container.java:2239)
	at java.awt.Component.dispatchEventImpl(Component.java:4899)
	at java.awt.Container.dispatchEventImpl(Container.java:2297)
	at java.awt.Component.dispatchEvent(Component.java:4721)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4904)
	at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4535)
	at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4476)
	at java.awt.Container.dispatchEventImpl(Container.java:2283)
	at java.awt.Window.dispatchEventImpl(Window.java:2746)
	at java.awt.Component.dispatchEvent(Component.java:4721)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:766)
	at java.awt.EventQueue.access$500(EventQueue.java:98)
	at java.awt.EventQueue$3.run(EventQueue.java:715)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:74)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:84)
	at java.awt.EventQueue$4.run(EventQueue.java:739)
	at java.awt.EventQueue$4.run(EventQueue.java:737)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:74)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:736)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:878)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:823)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:466)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:693)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:465)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:205)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:109)
	at java.awt.WaitDispatchSupport$2.run(WaitDispatchSupport.java:190)
	at java.awt.WaitDispatchSupport$4.run(WaitDispatchSupport.java:235)
	at java.awt.WaitDispatchSupport$4.run(WaitDispatchSupport.java:233)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.awt.WaitDispatchSupport.enter(WaitDispatchSupport.java:233)
	at java.awt.Dialog.show(Dialog.java:1077)
	at com.intellij.openapi.ui.impl.DialogWrapperPeerImpl$MyDialog.show(DialogWrapperPeerImpl.java:707)
	at com.intellij.openapi.ui.impl.DialogWrapperPeerImpl.show(DialogWrapperPeerImpl.java:432)
	at com.intellij.openapi.ui.DialogWrapper.doShow(DialogWrapper.java:1685)
	at com.intellij.openapi.ui.DialogWrapper.show(DialogWrapper.java:1644)
	at com.microsoft.azure.hdinsight.common.HDInsightHelperImpl$4.run(HDInsightHelperImpl.java:213)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.doRun(LaterInvocator.java:433)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.runNextEvent(LaterInvocator.java:416)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:399)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:764)
	at java.awt.EventQueue.access$500(EventQueue.java:98)
	at java.awt.EventQueue$3.run(EventQueue.java:715)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:74)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:734)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:878)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:827)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:466)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:704)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:465)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:205)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)

**Comment by jingyanjingyan** [comment]
Fix in build azure-toolkit-for-intellij-2019.2.4.develop.11072410.12-11-2019.zip (1323)\19.2&bundle.
Click Job and Storage account will get NPE error, track with #3814
